spec/models/user_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper'
 
 describe User do
+  it_behaves_like "has a uuid"
```

Comment:
I'm not sure about adding this to every spec. Seems a bit redundant to test behavior of a parent class on all the subclasses. I'd rather have an ApplicationRecord spec that tests it once. I will make this change since you're out.